### DISCUSSION TITLE
欢迎大家加入 TailwindCSS 群，答疑解惑、分享经验。

### DISCUSSION THREAD

**Discussion by tailwindchina** [tailwindchina/tailwindchina.com#1] [General]
欢迎大家加入 TailwindCSS 群，答疑解惑、分享经验。

![image](https://user-images.githubusercontent.com/76088906/102583666-0c007100-4140-11eb-8a2b-62df59867b8f.png)

注：申请时请填写备注：tailwindcss

**Comment by turkyden** [comment]
mark
